requirements.txt

```diff
@@ -2,14 +2,14 @@ aiodns==3.0.0
 aiohttp==3.8.1
 async_timeout==4.0.2
 atomicwrites-homeassistant==1.4.1
-attrs==22.1.0
+attrs==21.2.0
 awesomeversion==22.6.0
 brotli==1.0.9
 cchardet==2.1.7
 ciso8601==2.2.0
 colorlog==6.6.0
 cpe==1.2.1
-cryptography==37.0.4
+cryptography==36.0.2
```

Comment:
why do we downgrade?

Reply:
Because we were building a wheel in the CI and failing. According to https://wheels.home-assistant.io/musllinux/ we only have a wheel for `cryptography==36.0.2`

Reply:
Bumping `cryptography`: https://github.com/home-assistant/core/pull/76853

That should take care of it for now, however, should the SU work without wheels, but build wheels in case they are missing (like Core does?).

Reply:
Well, we build the wheel if not exists